bot-components/Git_utils.mli

```diff
@@ -39,12 +37,12 @@ val git_make_ancestor :
 val git_test_modified :
   base:string -> head:string -> string -> (bool, string) result Lwt.t
 
-val init_git_bare_repository : bot_info:Bot_components.Bot_info.t -> unit Lwt.t
+val init_git_bare_repository : bot_info:Bot_info.t -> unit Lwt.t
 
 val git_coq_bug_minimizer :
```

Comment:
It seems to me that this function and the next one are too specific to the case of Rocq to deserve to be moved in `bot-components`. WDYT?